pgml-extension/pgml_rust/src/backends/wrappers.py

```diff
@@ -0,0 +1,108 @@
+    "passive_aggressive_classification": sklearn.linear_model.PassiveAggressiveClassifier,
+    "ransac_regression": sklearn.linear_model.RANSACRegressor,
+    "theil_sen_regression": sklearn.linear_model.TheilSenRegressor,
+    "huber_regression": sklearn.linear_model.HuberRegressor,
+    "quantile_regression": sklearn.linear_model.QuantileRegressor,
+    "kernel_ridge_regression": sklearn.kernel_ridge.KernelRidge,
+    "gaussian_process_regression": sklearn.gaussian_process.GaussianProcessRegressor,
+    "gaussian_process_classification": sklearn.gaussian_process.GaussianProcessClassifier,
+    "svm_regression": sklearn.svm.SVR,
+    "svm_classification": sklearn.svm.SVC,
+    "nu_svm_regression": sklearn.svm.NuSVR,
+    "nu_svm_classification": sklearn.svm.NuSVC,
+    "linear_svm_regression": sklearn.svm.LinearSVR,
+    "linear_svm_classification": sklearn.svm.LinearSVC,
+    "ada_boost_regression": sklearn.ensemble.AdaBoostRegressor,
+    "ada_boost_classification": sklearn.ensemble.AdaBoostClassifier,
+    "bagging_regression": sklearn.ensemble.BaggingRegressor,
+    "bagging_classification": sklearn.ensemble.BaggingClassifier,
+    "extra_trees_regression": sklearn.ensemble.ExtraTreesRegressor,
+    "extra_trees_classification": sklearn.ensemble.ExtraTreesClassifier,
+    "gradient_boosting_trees_regression": sklearn.ensemble.GradientBoostingRegressor,
+    "gradient_boosting_trees_classification": sklearn.ensemble.GradientBoostingClassifier,
+    "hist_gradient_boosting_regression": sklearn.ensemble.HistGradientBoostingRegressor,
+    "hist_gradient_boosting_classification": sklearn.ensemble.HistGradientBoostingClassifier,
+    "random_forest_regression": sklearn.ensemble.RandomForestRegressor,
+    "random_forest_classification": sklearn.ensemble.RandomForestClassifier,
+}
+
+
+def estimator(algorithm_name, num_features, hyperparams):
+    return estimator_joint(algorithm_name, num_features, 1, hyperparams)
+
+
+def estimator_joint(algorithm_name, num_features, num_targets, hyperparams):
+    if hyperparams is None:
+        hyperparams = {}
+
+    def train(X_train, y_train):
+        instance = _ALGORITHM_MAP[algorithm_name](**hyperparams)
+
+        X_train = np.asarray(X_train).reshape((-1, num_features))
+
+        # Only support single value models for just now.
+        y_train = np.asarray(y_train).reshape((-1, num_targets))
+
+        instance.fit(X_train, y_train)
+        return instance
+
+    return train
+
+
+def test(estimator, X_test):
+    y_hat = estimator.predict(X_test)
+
+    # Single value models only just for now.
+    return list(np.asarray(y_hat).flatten())
+
+
+def predictor(estimator, num_features):
+    return predictor_joint(estimator, num_features, 1)
+
+
+def predictor_joint(estimator, num_features, num_targets):
+    def predict(X):
+        X = np.asarray(X).reshape((-1, num_features))
+        y_hat = estimator.predict(X)
+
+        # Only support single value models for just now.
```

Comment:
Can you make this a TODO, so it's easier to grep?

Reply:
It's in my head don't worry